ballerina-shell/modules/shell-cli/build.gradle

```diff
@@ -29,7 +29,7 @@ dependencies {
     implementation('org.jline:jline:3.11.0')
 
     testImplementation('org.testng:testng')
-    testImplementation('com.google.code.gson:gson')
+    implementation 'com.google.code.gson:gson:2.8.7'
```

Comment:
What's the need for specifying a specific version? Let's use the version used by others. This may cause multiple versions of the same library to be used here and there.

Reply:
This is the version used by others
`com.google.code.gson:gson:2.8.7`

Reply:
No, my point is, what's the need to specify the version here? This means to update the version, one has to keep track of multiple places.